src/client/src/containers/SelectFrameworks/styles.module.css

```diff
@@ -1,4 +1,4 @@
-.container {
+/* .container {
```

Comment:
Why are we commenting this out, do we still need it?

Reply:
I forgot to remove this - we don't need it anymore